plugins/languages/clangd_extensions.nix

```diff
@@ -0,0 +1,189 @@
+{
+  lib,
+  pkgs,
+  config,
+  ...
+}: let
+  helpers = import ../helpers.nix {inherit lib;};
+in
+  with lib; let
+    borderOpt = let
+      bordersTy =
+        types.enum ["double" "rounded" "single" "shadow" "none"];
+    in
+      helpers.defaultNullOpts.mkNullable (types.either bordersTy (types.listOf bordersTy))
+      ''"none"'' "";
+  in {
+    options.plugins.clangd_extensions = {
```

Comment:
Can you rename this to `clangd-extensions`? Keeps consistency with the rest

Reply:
Yeah I was not really sure to respect the case of the plugin or the other plugins, will do!